support(eslint): Adds support to EsLint using Red Style Guide

agawish opened the pull request:
# What I did
- Added 3 development dependencies (`eslint`, `eslint-config-red` and `eslint-plugin-import `)
  - The third one sometimes needed with `import` and `export` problems
- Added `.eslintrc.js` to configure EsLint to use Red Style Guide 
- Added another script `eslint` for easy access

# How I did it
- ran `npm install eslint eslint-config-red eslint-plugin-import --save-dev`
- ensured everything is fine by running `./node_modules/.bin/eslint random.js`
- ensured the same result by running `npm run eslint`

# How to verify it
- run `npm install`
- run `./node_modules/.bin/eslint random.js` to check and/or run `npm run eslint`

#  A picture of a cute animal
![animal-1850186_640](https://user-images.githubusercontent.com/87614/66126455-5cde8500-e5e1-11e9-8711-fa94d562cef9.jpg)

Fixes #2

GrosSacASac commented:
Thanks please put the eslint config inside package.json

GrosSacASac commented:
"eslint": "./node_modules/.bin/eslint random.js"
can be simplified into

    "eslint": "eslint random.js"

does it work without eslint-plugin-import ?

agawish commented:
Simplifying means that you assume `eslint` has been installed globally, so I didn't want to presume that.
It works without `eslint-plugin-import` my bad, I removed it from the dependencies and added the eslint config within `package.json`

GrosSacASac commented:
no, I don't have it installed globally, npm scripts just have access by default to everything inside .bin

GrosSacASac commented:
thanks

agawish commented:
Thank you, and sorry I remember I had some problems on certain OSes when not having the full path, feel free to change it.
